llvm/lib/ProfileData/Coverage/CoverageMappingReader.cpp

```diff
@@ -308,13 +309,21 @@ Error RawCoverageMappingReader::readMappingRegionsSubArray(
             return Err;
           if (auto Err = readIntMax(FID, std::numeric_limits<unsigned>::max()))
             return Err;
+          if (ID == 0)
+            return make_error<CoverageMapError>(
+                coveragemap_error::malformed,
+                "MCDCConditionID shouldn't be zero");
+          Params = CounterMappingRegion::MCDCBranchParameters{
+              (unsigned)ID, (unsigned)TID, (unsigned)FID};
```

Comment:
`static_cast`?

Reply:
I was so lazy since their casts would be removed by other PRs. I'll do shortly.